vignettes/strayr.Rmd

```diff
@@ -0,0 +1,154 @@
+---
+title: "StrayR"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{StrayR}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+
+
+## Under Construction
+
+
+## Using ABS structures
+
+Loading the package will lazily load the structures listed above. Call
+them with their name:
+
+```{r actual-load, include=FALSE}
+# use this to make sure README runs with current build
+# rather than installed version
+devtools::load_all()
+library(dplyr, warn.conflicts = FALSE)
```

Comment:
It looks like the problem I was having was related to [this issue in `absmapsdata`](https://github.com/wfmackey/absmapsdata/issues/20) where the new version of tibble requires the `sf` library to be explicitly loaded.

It ran fine for me if I added `library(sf)` to the `actual-load` block.
```suggestion
# use this to make sure README runs with current build
# rather than installed version
devtools::load_all()
library(sf)
library(dplyr, warn.conflicts = FALSE)
```

Reply:
That's a good pick-up, thanks @bryceroney .

Reply:
I'd encountered this same problem knitting the README, remember @wfmackey. Good to know what caused it.